red_queen/games/applications/grovers.py

```diff
@@ -0,0 +1,228 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the Apache 2.0 License.
+# See accompanying file /LICENSE for details.
+# ------------------------------------------------------------------------------
+
+# Copyright 2022 SRI-International organization
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     https://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# github repository:
+# https://github.com/Lementknight/QC-App-Oriented-Benchmarks/blob/master/grovers/qiskit/grovers_benchmark.py
+# Variable names from functions were modified to comply with pylint recommendations, and original benchmark
+# code was ommited as only grover's search algorithm's circuit construction was the only code needed
+"""
+Grover's Search Benchmark Program - Qiskit
+"""
+import os
+
+import pytest
+
+import numpy as np
+from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister
+from . import backends, run_qiskit_circuit
```

Comment:
Consider changing this into a package formatted import:
```suggestion
from red_queen.games.applications import backends, run_qiskit_circuit
```